project/backend/src/main/java/com/google/sps/data/JsonToRestaurantParser.java

```diff
@@ -17,7 +17,7 @@ private JsonToRestaurantParser() {}
   public static Restaurant toRestaurant(JSONObject body) throws JSONException {
     String id = body.getString("place_id");
     String name = body.getString("name");
-    String address = body.getString("formatted_address");
+    String address = body.getString("vicinity");
```

Comment:
I looked at some examples of the JSON returned by this new call and `vicinity` doesn't seem to be a full address. Is there anyway to request a fuller address using this new endpoint? `vicinity` doesn't seem to have the state and I think the address looks better when it has the state. If we cant change it, we can always get the address from the details call but just wondering..

Reply:
Hmm it doesn't look like the API returns that full address (check the link i sent a few days ago in our pod chat) but there is a field `plus_code.compound_code` that it looks like we can parse through to get the city if that is important

Reply:
I think vicinity has the city just not the state or zip code. I can just get it from the details request though. Was just wondering if it'd be possible to change the request to get more address info, but if not I'll just use the details data.

Reply:
Oh sorry I meant to say city/state/country. But I think it probably makes more sense for you to get it in details unless that causes any issues?